src/components/App.js

```diff
@@ -5,9 +5,6 @@ import ItemDescription from './ItemDescription';
 import '../App.css';
```

Comment:
- [ ] The `App` component is a top-level component. I recommend you take this file inside the `src` directory for a more logical document structure. :pray: